carmm/analyse/counterpoise_onepot.py

```diff
@@ -57,6 +57,10 @@ def counterpoise_calc(complex_struc, a_id, b_id, fhi_calc=None, a_name=None, b_n
     # Empty sites does not work with forces. Remove compute_forces.
     if 'compute_forces' in fhi_calc.parameters:
         fhi_calc.parameters.pop('compute_forces')
+    if 'sc_accuracy_forces' in fhi_calc.parameters:
+        print('Stop calculation as there is a convergence criterion regarding force.', '\n',
```

Comment:
Should this be an error, if you want the calculation to stop? Rather than a print statement?

Reply:
I thought it should not be an error. The function will return `None` (line 63) after printing the statement and no calculation will be done.

Reply:
Hmmm, but should a user be running BSSE with a force-evaluating calculation? If not, `None` could potentially return information that leaves a user thinking that the BSSE has run, when really it has none?

If the user shouldn't end up there, I think it should be an error. If instead the calculation should still work, then it is OK for `None`, in my mind.

Reply:
Yeah. We should not run BSSE with force-evaluation. I have changed this to raise an `KeyError` when `sc_accuracy_forces` is detected.